main/main.go

```diff
@@ -241,50 +243,7 @@ func runRelayer(
 	)
 
 	// Wait for logs from the subscribed node
```

Comment:
Can you update this comment to indicate that the following call will only return on error?